### ISSUE TITLE
More than 1 message besides call[0]

### ISSUE THREAD

**Issue by MattBlakeQA** [nils-hoyer/cypress-fail-on-console-error#11]
Hi, I'm just wondering why you only pass in `call[0]` to `isExcludedMessage`. Basically what I have noticed is that in my application it seems the entire error message is split between the first and second elements of `call`.

E.g.:

`call[0]: ERROR`
`call[1]: TypeError: Cannot set property 'status' of undefined`

The actual console error looks like:

`ERROR TypeError: Cannot set property 'status' of undefined`

But the console message getting checked against was simply `ERROR` and my config's excluded message of `Cannot set property...` was never being matched. Would it make sense to concatenate the strings? E.g. pass in `call.join(' ')`

**Comment by nils-hoyer** [comment]
Hi @MattBlakeQA,

what about ignoring the first error like this `excludeMessages: ['^ERROR$']` ?

can you please provide a minimal setup how to reproduce spy.args[0].length > 1. I was not able to reproduce that case.
What I get in my terminal is the following:

1. example test page:
```
<html>
    <script>
        try {
            let d = undefined;
            d[1];
        } catch(e) {
            console.error('ERROR ' + e)
        } finally {
            console.error('ERROR next message') 
        }
    </script>
</html>
````

2. state from sinon.spy: (call[] has length=1)
```
spy: {
    args: [
        [ // args[0][0] -> call[0]
            "ERROR TypeError: Cannot read property '1' of undefined"
        ],
        [ // args[1][0] -> call[0]
            "ERROR next message"
        ]
    ]
}
````

3. state from chrome console:

<img src="https://i.ibb.co/C9PTMWs/Bildschirmfoto-2021-04-16-um-13-56-53.png" alt="Bildschirmfoto-2021-04-16-um-13-56-53" border="0" width=50%>

**Comment by MattBlakeQA** [comment]
Hey @nils-hoyer,

I looked into this a bit more and it looks like the application is doing something like this. Using your example with an updated `console.error()` call:

`<html>
    <script>
        try {
            let d = undefined;
            d[1];
        } catch(e) {
            console.error('ERROR', e.message, 'and finally some other relevant information here')
        }
    </script>
</html>`

Note the multiple arguments to console.error(). This exception gets logged as `ERROR Cannot read property '1' of undefined and finally some other relevant information here`. In this specific example, there's 3 arguments for the call:

`call[0]: ERROR
call[1]: Cannot read property '1' of undefined
call[2]: and finally some other relevant information here`

**Comment by nils-hoyer** [comment]
Hi @MattBlakeQA,

Thank you very much for your feedback! Multiple console.error() parameters will now be handled properly. I used JSON.stringify() instead of join() to align with browser console output and cypress error message. 

fixed with version 2.1.0:
https://github.com/nils-hoyer/cypress-fail-on-console-error/releases/tag/2.1.0

commit hash: https://github.com/nils-hoyer/cypress-fail-on-console-error/commit/563c9880f75f52ede036fc987baf79948ed1f773
